plugins/core/webview/src/q-ui/components/login.vue

```diff
@@ -87,50 +65,20 @@ export default defineComponent({
         cancellable(): boolean {
             return this.$store.state.cancellable
         },
-        authorizationCode: {
-            get() {
-                return this.$store.state.authorizationCode
-            },
-            set(value: string | undefined) {
-                this.$store.commit('setAuthorizationCode', value)
-            }
-        },
-        isAuthorizationInProgress(): boolean {
-            return this.authorizationCode !== undefined
-        },
-        authenticatingText(): string {
-            if (this.selectedLoginOption?.id === LoginIdentifier.IAM_CREDENTIAL) {
-                return 'Connecting to IAM...'
-            } else if (this.selectedLoginOption?.id === LoginIdentifier.BUILDER_ID || this.selectedLoginOption?.id === LoginIdentifier.ENTERPRISE_SSO) {
-                return 'Authenticating in browser...'
-            }
-
-            return ''
-        },
-        requireConfirmationCodeOrNot(): boolean {
-            return this.selectedLoginOption?.requiresBrowser() === true && this.authorizationCode?.length !== 0
-        }
     },
     methods: {
         mutateStage(stage: Stage) {
             this.$store.commit('setStage', stage)
         },
         handleBackButtonClick() {
             if (this.cancellable) {
-                window.ideApi.postMessage({ command: 'toggleBrowser' })
+                window.ideApi.postMessage({command: 'toggleBrowser'})
             }
             this.mutateStage('START')
         },
         handleCancelButton() {
             window.ideClient.cancelLogin()
             this.mutateStage('START')
-            this.authorizationCode = undefined
-        },
-        changeTheme(darkMode: boolean) {
-            const oldCssId = darkMode ? "jb-light" : "jb-dark"
-            const newCssId = darkMode ? "jb-dark" : "jb-light"
-            document.body.classList.add(newCssId);
-            document.body.classList.remove(oldCssId);
```

Comment:
moved to root.vue

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/4370/files#diff-d831ab0cf73aa597f4bf0c976d638e38dc9abb52042ceea0bc6b6b81439b0bc9R43-R54